content/en/docs/configuration/device-provisioning-service.md

```diff
@@ -39,7 +39,7 @@ CoAP API as specified in the [workflow](./workflow.puml).
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
 | `apis.coap.address` | string | `Listen specification <host>:<port> for coap client connection.` | `"0.0.0.0:5688"` |
-| `apis.coap.protocols` | []string | `Protocol for coap connection. The supported values are: "tcp", "udp" .` | `["tcp"]` |
+| `apis.coap.protocols` | string | `Protocol for coap connection. The supported values are: "tcp", "udp" .` | `["tcp"]` |
```

Comment:
you can allow accept both protocols if you set `["tcp", "udp"]` - so change it to `[]string`

Reply:
fixed, thanks